.circleci/config.yml

```diff
@@ -71,7 +71,11 @@ jobs:
           version: 19.03.13
       - docker_login
       - run: echo "Releasing version" && ./gradlew printVersion
-      - run: ./gradlew publish
+      - run:
+          name: Publish to Maven Central
+          command: |
+            export ORG_GRADLE_PROJECT_signingKey=$(echo $SIGNING_KEY | base64 -d)
```

Comment:
In CCI, we generally just make the context variable itself named `ORG_GRADLE_PROJECT_signingKey` - doesn't really matter though if we're about to migrate.

Reply:
`ORG_GRADLE_PROJECT_signingKey` was present earlier in the context. the value is a multi-line. It did not work as expected. Hence, this workaround.
https://app.circleci.com/pipelines/github/hypertrace/javaagent/800/workflows/d4029b7a-9102-445f-a555-64b51a45ceb5/jobs/1982

Reply:
hmm strange - oh well, github doesn't expose them directly as env vars anyway so its moot.